android/app/src/main/java/io/peggyjo/peggymeter/database/MoodAdapter.java

```diff
@@ -79,6 +85,20 @@ public void addEntry(LogEntry entry) {
                 .build());
     }
 
+    public void editEntry(LogEntry entry, LogEntry localUpdateEntry, int position) {
```

Comment:
You can change the interface here to pass in the comment only - since it's the only field you are modifying. Then you should be able to edit the existing LogEntry inplace.
Thus you won't have to create a temporary LogEntry object.

Reply:
Yeah. I was trying to think about that.